vcbuild.bat

```diff
@@ -513,9 +525,15 @@ echo %cmd1%
 %cmd1%
 exit /b %ERRORLEVEL%
 
+:test-v8-exit
```

Comment:
weird, I thought I removed this and moved it to `test-v8.bat`. I will remove it. Sorry about that.